setup.py

```diff
@@ -103,6 +103,7 @@ def run_tests(self):
         'pygments>=2.2.0',
         'executing>=0.3.1',
         'asttokens>=2.0.1',
+        'backports.shutil-get-terminal-size==1.0.0',
```

Comment:
This should only be required in Python 2.

Reply:
Noted.